extensions/wikia/ARecoveryEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointMessage.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class ResourceLoaderAdEngineSourcePointMessage extends ResourceLoaderAdEngineSourcePointBase {
+	// increase this any time the local files change
+	const CACHE_BUSTER = 0;
+	const SCRIPT_URL = 'https://api.sourcepoint.com/script/msg?v=1';
+	const FALLBACK_SCRIPT_URL = __DIR__ . '/../SourcePoint/msg.js';
+
+	/**
+	 * Fallback data when request to external script fails
+	 * @return array ["script" => '', "modTitme" => '', "ttl" => '']
+	 */
+	protected function getFallbackDataWhenRequestFails() {
+		$scripts = [];
+		$scripts[] = (new ResourceLoaderScript())->setTypeLocal()->setValue(self::FALLBACK_SCRIPT_URL);
+		return $data = [
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$data" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)